athlinks-ontology.ttl

```diff
@@ -111,9 +111,9 @@ dc:date rdf:type owl:AnnotationProperty .
          rdfs:subClassOf :AthleticEventRole ;
          dc:creator "nextdude"^^xsd:string ;
          dc:date "2017-05-22T06:00:00Z"^^xsd:dateTime ;
-         rdfs:comment "A role that a person or sports team plays when they participate in an athletic event."@en ;
+         rdfs:comment "A role that an entry (such as a person or sports team) plays when they participate in an athletic event."@en ;
          rdfs:label "Athletic Event Competitor"@en .
-         # members:
+         # expected predicates:
          # schema:competitor -> individual (e.g. schema:Person) or team (e.g. schema:SportsTeam)
          # schema:sportsEvent -> athlinks:AthleticEvent
```

Comment:
This should be schema:event I think

Reply:
If it's an *Athletic* event competitor, then it seems reasonable to use the sports-specific predicate. But I don't care to argue the point too terribly much.